src/LoggerFactory.ts

```diff
@@ -15,7 +15,7 @@ export interface Event {
   timeStamp: number;
   params: any[];
   name: string;
-  level: string;
+  level: Lowercase<keyof typeof Level>;
```

Comment:
Has it been tested to link this package to `slf-debug` & `slf-sentry`, and verified that there are no type errors?

Reply:
Tested and no typing errors.